src/doc/rustc/src/codegen-options/index.md

```diff
@@ -355,21 +349,21 @@ Supported values for this option are:
 - `static` - non-relocatable code, machine instructions may use absolute addressing modes.
 
 - `pic` - fully relocatable position independent code,
-machine instructions need to use relative addressing modes.  
```

Comment:
Trailing white space in markdown is significant for formatting.

Reply:
I think it would be better to not rely on that behavior. Invisible markup is...hard to see. And I suspect this will happen again in the future.

I think using a trailing backslash would be better if explicit breaks are what you want.